src/components/column-sorting/column-sorting.component.ts

```diff
@@ -31,6 +31,17 @@ export class ColumnSortingComponent implements OnDestroy {
     /** Expose the sorting state enum to the view */
     columnSortingState = ColumnSortingState;
 
+    /** Determine which icon should be shown based on sort order */
+    get _icon(): string {
```

Comment:
Would it be better to assign an `_icon` property in the setter of `state` instead?